### DISCUSSION TITLE
2.3.10

### DISCUSSION THREAD

**Discussion by AmarnathCJD** [AmarnathCJD/gogram#92] [Announcements]
## What's Changed
* 32bit systems support by @aiexz in https://github.com/AmarnathCJD/gogram/pull/74
* refactor: types of function parameters can be combined by @deepsource-autofix in https://github.com/AmarnathCJD/gogram/pull/75
* refactor: unused parameter should be replaced by underscore by @deepsource-autofix in https://github.com/AmarnathCJD/gogram/pull/76
* fix: no cache check for `InputPeerChannel` & `InputPeerUser` by @aiexz in https://github.com/AmarnathCJD/gogram/pull/77
* Update react.go by @MukundSinghRajput in https://github.com/AmarnathCJD/gogram/pull/91

## New Contributors
* @MukundSinghRajput made their first contribution in https://github.com/AmarnathCJD/gogram/pull/91

**Full Changelog**: https://github.com/AmarnathCJD/gogram/compare/2.3.6...2.3.10

<hr /><em>This discussion was created from the release <a href='https://github.com/AmarnathCJD/gogram/releases/tag/2.3.10'>2.3.10</a>.</em>

**Comment by YHlorra** [comment]
test
